Blankmon/src/application/BaseAI.java

```diff
@@ -1,63 +1,79 @@
 package application;
+
 import java.util.ArrayList;
 import java.util.Random;
 import application.enums.*;
 
-public class BaseAI {
-	public BaseAI() {
+public class BaseAI
+{
+	public final String useItem(ArrayList<Item> items, Anature currentAnature, int healthThreshold)
```

Comment:
What was the reason for using final in this class?

Reply:
To ensure that all Ai use the same logic for item consumption. Since it would be confusing if one ai used items differently then others.